src/ui/AccountDropdown/index.svelte

```diff
@@ -38,41 +39,58 @@
         <a
           class="btn-ghost"
           href="https://app.santiment.net/alerts?tab=1"
-          on:click={window.__onLinkClick}>My alerts</a
-        >
+          on:click={window.__onLinkClick}>My alerts</a>
         <a class="btn-ghost" href="https://app.santiment.net/assets" on:click={window.__onLinkClick}
-          >My watchlists</a
-        >
+          >My watchlists</a>
         <a
           class="btn-ghost"
           href="https://insights.santiment.net/my"
-          on:click={window.__onLinkClick}>My insights</a
-        >
+          on:click={window.__onLinkClick}>My insights</a>
         <a
           href="https://insights.santiment.net/new"
           class="write btn-1 btn--s"
-          on:click={window.__onLinkClick}>Write insight</a
-        >
+          on:click={window.__onLinkClick}>Write insight</a>
       </section>
       <hr />
     {/if}
 
     <section>
+      {#if !currentUser}
+        <a href="/login" class="loginbtn" on:click={window.__onLinkClick}>
+          <Svg id="user" w="16" class="mrg-s mrg--r" />
+          <span>Log in</span>
+        </a>
+      {/if}
+
+      {#if appVersionState !== undefined && appVersionState >= 0}
+        <div class="appversion">
```

Comment:
This section should be rendered inside `<section>` which surrounded by `<hr>` elements